.github/ISSUE_TEMPLATE/package.yml

```diff
@@ -0,0 +1,60 @@
+name: Packages
+description: Report problems and suggest ideas for the packages
+labels: [package]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `package` label][package-issues] (including
+        closed issues) before you continue.
+
+
+        [package-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Apackage+is%3Aopen
+
+  - type: dropdown
+    attributes:
+      label: Kind of issue
+      description: >-
+        If your issue type is not here, select "other" and explain in the
+        "Description" field below.
+      options: [improvement, bug, other]
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Which package does this issue concern?
+      options: [NPM, Packagist]
+      multiple: true
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Anything relevant, for example:
+        - For bugs: "Steps to reproduce" and "Expected behavior"
+        - For improvements: An example of a use case
+        - Package issues: Version information
+          - For JavaScript/NodeJS: `node --version`
+          - For PHP/Packagist: `composer --version`
```

Comment:
How about just:

```suggestion
        - Software versions (NodeJS & NPM, PHP & Packagist, Browser)
```

Reply:
A separate input would be better because can we require it, I'll add one.

Reply:
On second thought, I don't think a separate input should be used here. Not all package issues will need version information and if we make an optional input, GitHub will render it with "No response" (I believe we already discussed this).

Reply:
I think I'd like a separate input better. I think version information (or which browser is being used) is almost always useful. I would be okay with keeping it optional (and the corresponding "No response" output), or making it required and adding a note like `[...] If you are unsure, enter "None"`.

Additionally, we should probably consider adding an input for the version of the package that is being used :thinking:

Reply:
Added in the latest commit, unsure on the wording though.